server/admin/server.go

```diff
@@ -150,6 +154,10 @@ func NewServer(cfg *SrvConfig) (*Server, error) {
 			rm.Get("/forgive_match/{"+matchIDKey+"}", s.apiForgiveMatchFail)
 			rm.Post("/notify", s.apiNotify)
 		})
+		r.Route("/asset/{"+assetSymKey+"}", func(rm chi.Router) {
+			rm.Get("/", s.apiAsset)
+			rm.Post("/", s.apiAssetPOST)
```

Comment:
How about `/api/setfeescale`? We're only setting one field of the struct returned in the GET response.

Reply:
Might as well.  I was anticipating a need to change other asset parameters such as maxFeeRate, lotsize, ratestep, etc. but on reflection, I believe those hot changes would be done via editing markets.json and signalling to reload/parse it.